README.md

```diff
@@ -242,6 +242,22 @@ exports.onCameraMove = onCameraMove;
 ```
 
 ## Custom Info Windows (Beta)
+>[!WARNING]
+> if you are using NS7 `infoWindow Template` won't work from a xml file! a temporary solution will be declaring infoWindow Template from the code behind like this :
```

Comment:
Hi @kefahB, your comment made me remember that in my ns-angular application, I had to load the info template by :

1. First define the template in a variable
```javascript
const infoWindowTemplate = `<StackLayout orientation="vertical" width="240" height="340">
        	                <Label text="{{title}}"    marginTop="26"  marginLeft="20" textWrap="true" color="black"  fontSize="18"></Label>
                                <Label text="{{address}}"  marginLeft="20" textWrap="true" color="gray"    fontSize="14"></Label>
                            </StackLayout>`;
```
2. Assign the template to each marker 
```typescript
    loadMarkers() {

        this.testMarkers.forEach(m => {

            if (m.type !== this.pickupChannel) {
                console.log(`Skipping this marker: ${m.type} != ${this.pickupChannel}`);
                return;
            }

            const marker              = new Marker();
            marker.infoWindowTemplate = infoWindowTemplate;
            marker.position           = Position.positionFromLatLng(m.latitude, m.longitude);
            marker.title              = m.name;
            marker.snippet            = `${m.zip} ${m.address} - ${m.city}`;

            console.log(`Adding new marker to the map: ${marker.title};`);
            this.mapView.addMarker(marker);
        });

    }
``` 

It was before of NS7 upgrade. Maybe it can help you!

Reply:
Hi,

It is ont needed to assign the template to the markers directly .. I've add new property to `map-view-common.ts` .. you can declare the template on the `mapView` directly .. Ill push the commit right now ;)